src/handler/getPersonById/index.ts

```diff
@@ -1,27 +1,19 @@
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
 
-import { getPersonById } from '../../pseudoDB/index';
+import { getPersonById } from '../../services/person';
 
 export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
-  if (event.pathParameters && event.pathParameters.id && isNaN(Number(event.pathParameters.id))) {
-    return {
-      statusCode: 400,
-      body: JSON.stringify({ code: 400, error: "The 'id' parameter must be a number." }),
-    };
-  }
+  const id = event.pathParameters && event.pathParameters.id ? event.pathParameters.id : '0';
```

Comment:
Uso de ternário deixa mais difícil a lógica mais complexa. E nesse caso vc deveria retornar um erro 4xx pela falta do parâmetro.

Reply:
Nesse caso nunca vai cair no '0', entra naquele mesmo caso que se estiver sem o 'id' no path nem vai cair nessa função. Eu fiz isso porque a função que pega o registro espera uma string e se eu não fizer assim o esling fica dando que esse event.pathParameters.id pode ser 'null', mas na verdade ele nunca vai ser 'null'.

Reply:
O que eu poderia fazer nesse caso é permitir a função que busca o registro no DB receber null e dar um erro caso o valor seja null

Reply:
Mesmo assim seria um código sem uso, nunca iria cair nesse if

Reply:
Acho que se vc colocar essa lógica dentro de um if != null ficaria mais claro. Pq esse ternário de 0 tbem eh sem uso e causa uma certa confusão pra quem passa o olho.

Reply:
![image](https://user-images.githubusercontent.com/7038499/192035025-eb058721-1c13-4b8c-b960-cad17522adc7.png)

Vai dar quase na mesma, ainda vou ter que iniciar o id com uma string, caso contrário ela vai ser 'undefined' e vou receber um erro do linter quando passo o 'id' para a outra função: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
E também recebo um erro que não consegui resolver dentro do if que é fazer 'object destructuring' :  Use object destructuring.eslint[prefer-destructuring]

Reply:
Consegui resolver essa questão em apenas uma linha, vou marcar como resolvido.